src/contribution/contribution.jl

```diff
@@ -0,0 +1,61 @@
+module Contribution
+
+using ..TypeDBClient
+
+g = TypeDBClient
```

Comment:
You can probably do this?
```julia
import ..TypeDBClient as g
```

Reply:
That's not allowed in 1.6 or 1.7

Reply:
What do you mean... It seems ok to me? If not, you can at least make `g` a const.

```julia
julia> module Foo
           foo() = 1
           module Bar
               import ..Foo as g
               bar() = g.foo()
           end
       end
Main.Foo

julia> Foo.Bar.bar()
1
```

Reply:
I tried your suggestion and get and error if I do so. I make it to a constant value